src/test/java/org/folio/rest/impl/RestVerticleTest.java

```diff
@@ -69,38 +68,28 @@
   private JsonObject templateObject2 = templateObject
     .put("description", NEW_TEMPLATE_DESCRIPTION);
 
-  @Before
-  public void setUp(TestContext context) throws IOException {
+  @BeforeClass
+  public static void setUpClass(TestContext context) throws IOException {
     int okapiPort = NetworkUtils.nextFreePort();
     mockServerPort = NetworkUtils.nextFreePort();
 
-    WireMockServer wireMockServer = new WireMockServer(mockServerPort);
+    wireMockServer = new WireMockServer(mockServerPort);
     wireMockServer.start();
-    wireMockServer.stubFor(
-      get(urlPathEqualTo("/patron-notice-policy-storage/patron-notice-policies"))
-        .willReturn(okJson(new JsonObject()
-          .put("patronNoticePolicies", new JsonArray())
-          .put("totalRecords", 0).encode())));
+    setupStub();
 
     Async async = context.async();
-    TenantClient tenantClient = new TenantClient("localhost", okapiPort, "diku", "diku");
     vertx = Vertx.vertx();
     okapiUrl = "http://localhost:" + okapiPort;
     templateUrl = okapiUrl + "/templates";
 
+    PostgresClient.getInstance(vertx).startEmbeddedPostgres();
```

Comment:
All code that configures and starts and stops the database for testing should be removed. PostgresClient was updated to do this automatically: https://github.com/folio-org/raml-module-builder#postgresql-integration